lab-jr/.gitignore

```diff
@@ -0,0 +1 @@
+node_modules
```

Comment:
Anything else you might want to add to this file? This is a great start, but there's always more! :-)